docs/integrations/azureBlobstorage/locations.md

```diff
@@ -0,0 +1,26 @@
+---
+id: locations
+title: Azure BlobStorage Locations
+sidebar_label: Locations
+# prettier-ignore
+description: Integrating source code stored in BlobStorage Storage into the Backstage catalog
+---
+
+The Azure BlobStorage integration supports loading catalog entities from Azure
+BlobStorage. Entities can be added to
+[static catalog configuration](../../features/software-catalog/configuration.md),
+or registered with the
+[catalog-import](https://github.com/backstage/backstage/tree/master/plugins/catalog-import)
+plugin.
+
+```yaml
+integrations:
+  azureBlobStorage:
+    - accountName: account123
+      secretAccessKey: ${AZURE_BLOB_SAS_TOKEN}
+```
+
+The configuration is a structure with two elements:
+
+- `accountName`: The BlobStorage host; only `{accountName}.blob.core.windows.net` is supported.
+- `secretAccessKey` (optional): A SAS token; including the leading `?`.
```

Comment:
Is this a "Shared Access Key" or a "Shared Access Signature"?  The property's called `secretAccessKey`, but based on the description it sounds more like a shared access signature.

* Access Key: `oh6I9j7GbWHYmJsrnXNMzuem+EkduKh3NkGfY7RAr/IP0B+Tudv9B5/xYKSkOEDI4ZsivGGDlFkl+ASt3YqXFw==`
* Shared Access Signature (Referred to as SAS Token in the Azure Portal): `?sv=2022-11-02&ss=bfqt&srt=s&sp=rwdlacupiytfx&se=2023-05-09T01:43:39Z&st=2023-05-08T17:43:39Z&spr=https&sig=REDACTED`.